.github/workflows/ci.yaml

```diff
@@ -59,6 +59,14 @@ jobs:
           test
           dynamodbLocalTestCleanup
           stopDynamodbLocal
+      - name: test without coverage
+        if: matrix.version == '3.2.0'
```

Comment:
NOTE: IIRC, Scala 3.2 compiler supports code coverage.

https://dotty.epfl.ch/docs/internals/coverage.html

Reply:
I'm investigating how to use this feature.